server/plugin/api.go

```diff
@@ -331,7 +331,11 @@ func (p *MatterpollPlugin) handleCreatePoll(_ map[string]string, request *model.
 			"poll_id": poll.ID,
 		},
 	}
+
 	model.ParseSlackAttachment(post, actions)
+	if poll.Settings.ShowVoters {
+		post.AddProp("card", poll.ToCard(p.bundle, p.ConvertCreatorIDToDisplayName))
```

Comment:
I would like to use `p.ConvertUserIDToDisplayName` instead of `p.ConvertCreatorIDToDisplayName` here, because it make a card more informative.
<img width="468" alt="スクリーンショット 2022-06-16 23 01 15" src="https://user-images.githubusercontent.com/1453749/174098421-94887fe6-395f-43c0-a132-eaa6b50c3838.png">

Reply:
I'm on it.

Reply:
Done